openassessment/templates/openassessmentblock/staff_area/oa_student_info.html

```diff
@@ -0,0 +1,318 @@
+                    <span>{% trans "Peer Assessments Completed by This Learner" %}</span>
+                </h2>
+                <div class="ui-toggle-visibility__content">
+                    {% if submitted_assessments %}
+                        {% for assessment in submitted_assessments %}
+                            {% with peer_num=forloop.counter %}
+                                <h4 class="title--sub">{% trans "Assessment" %} {{ peer_num }}:</h4>
+                                <table class="staff-info__status__table" summary="{% trans "Assessment" %}">
+                                    <thead>
+                                    <tr>
+                                        <th abbr="Criterion" scope="col">{% trans "Criterion" %}</th>
+                                        <th abbr="Selected Option" scope="col">{% trans "Selected Option" %}</th>
+                                        <th abbr="Feedback" scope="col">{% trans "Feedback" %}</th>
+                                        <th abbr="Points" scope="col">{% trans "Points" %}</th>
+                                        <th abbr="Points Possible" scope="col">{% trans "Points Possible" %}</th>
+                                    </tr>
+                                    </thead>
+
+                                    <tbody>
+                                    {% for criterion in rubric_criteria %}
+                                        {% for part in assessment.parts %}
+                                            {% if part.option.criterion.name == criterion.name %}
+                                                <tr>
+                                                    <td class="label">{{ criterion.label }}</td>
+                                                    <td class="value">{{ part.option.label }}</td>
+                                                    <td class="value">{{ part.feedback }}</td>
+                                                    <td class="value">{{ part.option.points }}</td>
+                                                    <td class="value">{{ criterion.total_value }}</td>
+                                                </tr>
+                                            {% endif %}
+                                        {% endfor %}
+                                    {% endfor %}
+                                    </tbody>
+                                </table>
+                                <h4 class="title--sub">{% trans "Overall Feedback" %}</h4>
+                                <div class="student__answer__display__content">
+                                    {{ assessment.feedback|linebreaks }}
+                                </div>
+                            {% endwith %}
+                        {%  endfor %}
+                    {% else %}
+                        <p>{% trans "This learner has not completed any peer assessments." %}</p>
```

Comment:
@andy-armstrong I'd also like to suggest a minor tweak here: "This learner has not performed any peer assessments."